homeassistant/components/surepetcare/__init__.py

```diff
@@ -51,6 +54,7 @@
                     cv.ensure_list, [cv.positive_int]
                 ),
                 vol.Optional(CONF_PETS): vol.All(cv.ensure_list, [cv.positive_int]),
+                vol.Optional(CONF_TIMEOUT, default=15): cv.positive_int,
```

Comment:
Maybe we don't need the option but can just set the default timeout to 15 seconds? We try to avoid adding options for timeout.

Reply:
As this was the second time, I choose this way :D